js/dir_test.ts

```diff
@@ -0,0 +1,43 @@
+import { test, testPerm, assert, assertEqual } from "./test_util.ts";
+import * as deno from "deno";
+
+test(function NotNullcwd() {
+  assert(deno.cwd() != null);
+});
+
+testPerm({ write: true }, function test_cwd_output() {
+  const path = deno.makeTempDirSync() + "/dir/subdir";
+  deno.mkdirSync(path);
+  deno.chdir(path);
+  assertEqual(deno.cwd(), path);
```

Comment:
The tests are guaranteed to run in the source directory - so I think you should be able to do these tests without using makeTempDir - that would make them more decoupled from other machinery